include/AssembleElemSolverAlgorithm.h

```diff
@@ -96,6 +96,16 @@ class AssembleElemSolverAlgorithm : public SolverAlgorithm
 
         SharedMemData<DeviceTeamHandleType, DeviceShmem> smdata(
           team, nDim, dataNeededNGP, nodesPerEntity, rhsSize);
+        for (int i = 0; i < rhsSize; i++) {
```

Comment:
Would it be better to use something like `set_vals(smdata.rhs, 0.0);` ? We do that in `AssembleEdgeSolverAlgorithm.h`

Reply:
Good point, will fix